tests/script-based-pre/stubbing_compiler_sessions/check_compiler_sessions.expected

```diff
@@ -1,3 +0,0 @@
-Complete - 6 successfully verified harnesses, 0 failures, 6 total.
```

Comment:
Why can we safely delete all these test-related files?

Reply:
I can get the test back if you want.

Reply:
I'll move the test itself to the `expected` suite, since it's now just a regular test to ensure we can verify harnesses with different sets of stubs.